core/src/klio_core/config/_io.py

```diff
@@ -350,6 +350,25 @@ class KlioReadAvroEventConfig(KlioEventInput, KlioReadAvroConfig):
     pass
 
 
+@attr.attrs(frozen=True)
+class KlioWriteAvroConfig(KlioAvroConfig):
+    # Either file_path_prefix or location must be set
+    # Right now `schema` is also allowed to pass inbut anything other than
+    # the default schema cannot make proper use of `write_record`
+    file_path_prefix = attr.attrib(type=str, default=None)
+    location = attr.attrib(type=str, default=None)
+    codec = attr.attrib(type=str, default="deflate")
+    file_name_suffix = attr.attrib(type=str, default="")
+    num_shards = attr.attrib(type=int, default=0)
```

Comment:
Think you missed one - [shard_name_template](https://github.com/spotify/klio/blob/master/lib/src/klio/transforms/io.py#L441)